### ISSUE TITLE
CAPI: Missing GetKeyIdentifierFromVirtualKeyCode equivalent

### ISSUE THREAD

**Issue by chances** [ultralight-ux/Ultralight#290]
The C API surface is missing an equivalent function to map virtual key codes to key identifiers.

Also, the [W3C link in `KeyEvent.h`](https://github.com/ultralight-ux/Ultralight-API/blob/master/Ultralight/KeyEvent.h#L132) is broken.

Here is a working link: https://www.w3.org/TR/2003/WD-DOM-Level-3-Events-20030331/keyset.html

**Comment by adamjs** [comment]
Thanks for the heads up-- I'll add this to the C API

**Comment by adamjs** [comment]
I've fixed the link in documentation-- as for the missing `GetKeyIdentifierFromVirtualKeyCode` it's actually not necessary in the C API since it's already called for you in `ulCreateKeyEvent()` (I should document this behavior better), here's the relevant source:

```cpp
ULKeyEvent ulCreateKeyEvent(ULKeyEventType type, unsigned int modifiers,
  int virtual_key_code, int native_key_code, ULString text, ULString unmodified_text,
  bool is_keypad, bool is_auto_repeat, bool is_system_key) {

  KeyEvent* evt = new KeyEvent();
  evt->type = (KeyEvent::Type)type;
  evt->modifiers = modifiers;
  evt->virtual_key_code = virtual_key_code;
  evt->native_key_code = native_key_code;
  evt->text = ToString(text);
  evt->unmodified_text = ToString(text);
  evt->is_keypad = is_keypad;
  evt->is_auto_repeat = is_auto_repeat;
  evt->is_system_key = is_system_key;
  if (evt->type != KeyEvent::kType_Char)
    GetKeyIdentifierFromVirtualKeyCode(evt->virtual_key_code, evt->key_identifier);

  return (ULKeyEvent)evt;
}
```

**Comment by chances** [comment]
Ah, thank you. I thought perhaps having this in the CAPI would fix #65, but I guess that's not the issue.
